### ISSUE TITLE
specific transmitter ID

### ISSUE THREAD

**Issue by michaeldoll** [enocean-js/enocean-js#252]
Hello,
I have a shutter actuator that I can't get to easily.  On the actuator is already a button learned. I would like to use this button-Id to control the actuator via node-Red. 
can you set the actuator node to output a specific transmitter id ?

**Comment by Holger-Will** [comment]
yes and no, you can change the base id of your enocean USB stick 10 times. so basically you can set the base to something in the same range as your tought in button and do mimic that exact button. but you are stuck with this range and can not chage back anth forth your base id to mimic multiple different buttons. It's a hack that works for this one specific case.
